internal/ui/backup/json.go

```diff
@@ -208,7 +208,7 @@ type statusUpdate struct {
 
 type errorUpdate struct {
 	MessageType string `json:"message_type"` // "error"
-	Error       error  `json:"error"`
+	Error       string `json:"error"`
```

Comment:
Based on your Go playground test, sounds like there's not much value in trying to be backwards compatible here?

i.e. I might as well leave this part of the PR as-is and just re-use the "error" field as a string instead of an an object.

Reply:
I think it's the other way around. My go playground example shows that if we change `error` to `string`, then it will be a breaking change. Whereas the inline struct variant remains backwards compatible. Having a nested struct isn't entirely ideal, but could be worse. So let's follow the approach of turning the error field into a struct.

For consistency that should also apply to the `restore` command.

Reply:
Got it, will do :+1:

Reply:
Done.